docs/minimal/minimal.md

```diff
@@ -15,3 +15,23 @@ Minimal downtime migrations are only supported for MySQL, Postgres and Oracle so
 A minimal downtime migration consists of two components, migration of existing data from the database and the stream of changes (writes and updates) that are made to the source database during migration, referred to as change database capture (CDC). Using Spanner migration tool, the entire process where Datastream reads data from the source database and writes to a GCS bucket and data flow reads data from GCS bucket and writes to spanner database can be orchestrated using a unified interface. Performing schema changes on the source database during the migration is not supported. This is the suggested mode of migration for most databases.
 
 ![](https://services.google.com/fh/files/helpcenter/asset-ripjb7eowf.png)
+
+## Sharded migrations
+
+{: .note }
+Sharded migrations are currently only supported for MySQL.
+
+Spanner migration tool supports sharded migrations for MySQL. Spanner migration tool does this is my multiplexing a minimal downtime migration across multiple shards. It applies the user configured schema uniformly across each shard, and provides an integrated experience to perform an end-to-end sharded migration. Below is the architecture of how sharded migrations work:
```

Comment:
type: by multiplexing

Reply:
It does not really apply the schema on the shard. It applies it on the Spanner schema.

Reply:
Thanks, the language was not clear. I have rephrased it to be more clearer. Also corrected typo